app/assets/javascripts/account/stack_theme/core/app.js

```diff
@@ -0,0 +1,327 @@
+            var block_ele = $(this).closest('.card');
+
+            // Block Element
+            block_ele.block({
+                message: '<div class="ft-refresh-cw icon-spin font-medium-2"></div>',
+                timeout: 2000, //unblock after 2 seconds
+                overlayCSS: {
+                    backgroundColor: '#FFF',
+                    cursor: 'wait',
+                },
+                css: {
+                    border: 0,
+                    padding: 0,
+                    backgroundColor: 'none'
+                }
+            });
+        });
+
+        // Close Card
+        $('a[data-action="close"]').on('click',function(){
+            $(this).closest('.card').removeClass().slideUp('fast');
+        });
+
+        // Match the height of each card in a row
+        setTimeout(function(){
+            $('.row.match-height').each(function() {
+                $(this).find('.card').not('.card .card').matchHeight(); // Not .card .card prevents collapsible cards from taking height
+            });
+        },500);
+
+
+        $('.card .heading-elements a[data-action="collapse"]').on('click',function(){
+            var $this = $(this),
+            card = $this.closest('.card');
+            var cardHeight;
+
+            console.log(parseInt(card[0].style.height,10));
+            if(parseInt(card[0].style.height,10) > 0){
+                cardHeight = card.css('height');
+                card.css('height','').attr('data-height', cardHeight);
+            }
+            else{
+                if(card.data('height')){
+                    cardHeight = card.data('height');
+                    card.css('height',cardHeight).attr('data-height', '');
+                }
+            }
+        });
+
+        //Adjust fixed header body top padding
+        /*if(!$('.header-navbar').hasClass('navbar-fixed-top')){
+            $('body').css('padding-top',0);
+        }*/
+        // Remove the top padding in
+        /*if(! $('.header-navbar').hasClass('navbar-fixed-top')){
+            $('body').css('padding','0');
+        }*/
+
+        // Add open class to parent list item if subitem is active except compact menu
+        var menuType = $body.data('menu');
+        if(menuType != 'horizontal-menu'){
+            $(".main-menu-content").find('li.active').parents('li').addClass('open');
+        }
+        if(menuType == 'horizontal-menu'){
+            $(".main-menu-content").find('li.active').parents('li:not(.nav-item)').addClass('open');
+            $(".main-menu-content").find('li.active').parents('li').addClass('active');
+        }
+
+        //card heading actions buttons small screen support
+        $(".heading-elements-toggle").on("click",function(){
+            $(this).parent().children(".heading-elements").toggleClass("visible");
+        });
+
+        //  Dynamic height for the chartjs div for the chart animations to work
+        var chartjsDiv = $('.chartjs'),

```

Comment:
remove this
and your chart from dashboard.
and all JS what you have for charts.
We will add them back in next PR